### PR TITLE
DEV-1124: Sandbox testing guide layout changes

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Break out into separate pages
- Add CTAs and links within the guide
- Update links that reference `/guide/sandbox/*`

### CODE REVIEW

File: pages/guides/sandbox/customers.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+The [Sandbox Dashboard](https://dashboard-sandbox.dwolla.com) allows you to manage Customers, as well as transfers associated with the Customers that belong to your Sandbox account. Once your application has [created its Customers](https://developers.dwolla.com/api-reference/customers/create), you can access the [Sandbox Dashboard](https://dashboard-sandbox.dwolla.com) to validate that the request was recorded properly in our test environment.
+
+<InlineCTA
+  icon={conceptsIcon}
+  text="There are multiple Customer types within the Dwolla API. Use our concept article for a more in-depth overview of each Customer type and its capabilities."
+  href="/concepts/customer-types"
+/>
+
+##### **Verified Customers**
```
Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: I'd say let's remove this header. I think context around the customer type is implied as well as included within the sections below this

File: pages/guides/sandbox/customers.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+import imageFail from "../../../assets/images/content-images/test-document-upload-fail.png";
+import imageSuccess from "../../../assets/images/content-images/test-document-upload-success.png";
+import conceptsIcon from "../../../assets/images/content-images/content-icons/concepts-icon.svg";
+import guidesIcon from "../../../assets/images/content-images/content-icons/guides-icon.svg";
+
+# Testing Customers
+
+#### Manage Customers
```
Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: Should we change this to an h3 and title it `Manage Customers in the Dashboard` ?

File: pages/guides/sandbox/customers.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+<InlineCTA
+  icon={conceptsIcon}
+  text="There are multiple Customer types within the Dwolla API. Use our concept article for a more in-depth overview of each Customer type and its capabilities."
+  href="/concepts/customer-types"
+/>
+
+##### **Verified Customers**
+
+## Simulate identity verification statuses
```
Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: Change to h3 for aesthetics

File: pages/guides/sandbox/customers.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+With business Verified Customers, documents may be needed for the Controller, the business, or both the Controller and business. To simulate the `document` status for the controller, submit `document` in the **controller firstName** parameter.
+Similarly, to simulate the `document` status for the business, submit `document` in the **businessName** parameter. To simulate both the Controller and the business in `document`, submit `document` in both the **controller firstName** and the **businessName** parameters.
+
+## Simulate KBA verified and failed events
```
Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: Change to h3 for aesthetics

File: pages/guides/sandbox/customers.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  icon={guidesIcon}
+  text="More information on KBA status for Personal Verified Customers and the related endpoints."
+  href="/guides/personal-verified-customer/handle-verification-statuses#handling-status-kba"
+/>
+
+To simulate the `customer_kba_verification_passed` event as the result of KBA success in Sandbox, answer all four questions with either “None of the above” or “I have never been associated with this vehicle”. As a result, the Customer will be placed in a verified status and the `customer_verified` event is triggered.
+
+To simulate the `customer_kba_verification_failed` event as the result of KBA failure in Sandbox, answer the questions with any answer choices other than “None of the above” or “I have never been associated with this vehicle”. As a result, the Customer will be placed in a document status and the `customer_verification_document_needed` event is triggered.
+
+## Simulate document upload approved and failed events
```
Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: Change to h3 for aesthetics

Comment (spencerhunter) [inline pages/guides/sandbox/customers.mdx]: Change to h3 for aesthetics

**Review by spencerhunter — DISMISSED** [review]

File: pages/guides/sandbox/customers.mdx (hunk 1 above)
Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: That makes sense! 

Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

File: pages/guides/sandbox/customers.mdx (hunk 3 above)
Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

File: pages/guides/sandbox/customers.mdx (hunk 5 above)
Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

File: pages/guides/sandbox/customers.mdx (hunk 4 above)
Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

File: pages/guides/sandbox/customers.mdx (hunk 2 above)
Comment (ShreyaThapa) [reply pages/guides/sandbox/customers.mdx]: Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**